common/lib/xmodule/xmodule/modulestore/xml_exporter.py

```diff
@@ -264,8 +264,14 @@ def process_extra(self, root, courselike, root_courselike_dir, xml_centric_cours
             'about', 'about', '.html'
         )
 
+        course_policy_dir_name = courselike.location.run
+        if courselike.url_name != courselike.location.run and courselike.url_name == 'course':
+            # Use url_name for split mongo because course_run is not used when loading policies.
```

Comment:
Nit: It'd be nice if this comment explained how one knows that this course is a split course by using the above check.